packages/browser/src/client/client.ts

```diff
@@ -11,7 +11,7 @@ export const HOST = [location.hostname, PORT].filter(Boolean).join(':')
 export const SESSION_ID
   = PAGE_TYPE === 'orchestrator'
     ? getBrowserState().contextId
-    : crypto.randomUUID()
+    : getBrowserState().testerContextId
```

Comment:
let's call it `testerId` instead - too much `context` bloat I feel like

Reply:
done